### DISCUSSION TITLE
有没有办法实现类似css中box-shadow: inset 0px 0px 480px 80px rgba(3,24,52,0.92); 内阴影的效果呢

### DISCUSSION THREAD

**Discussion by klousCan** [BYJRK/DotNet-Discussions#132] [WPF 开发]
box-shadow: inset 0px 0px 480px 80px rgba(3,24,52,0.92);
<img width="1240" alt="24db3a208435bae67d0488d523f7fb1" src="https://github.com/BYJRK/DotNet-Discussions/assets/9686646/205b19d3-a1ab-4c49-b81d-b7b7f66ce2fb">

**Comment by BYJRK** [comment]
参考一下这篇文章：https://zhuanlan.zhihu.com/p/457403040
